packages/cli/utils/installDependencies.ts

```diff
@@ -27,6 +39,12 @@ export const installDependencies = async ({ component, outputPath, packageManage
     const componentsPrompt = promptsMap[PromptsNames.ShouldCopyComponents](components);
     const { shouldCopyComponents } = await prompts(componentsPrompt, { onCancel: () => process.exit(0) });
 
-    if (shouldCopyComponents) await copyComponents(componentsDependencies, outputPath);
+    if (shouldCopyComponents)
```

Comment:
I know it's not part of this PR but should not there be an attempt to install dependencies after the component copy too?

Reply:
Good point. I have updated the copyComponents file and added the function to install dependencies as it works for the base component.